src/app.js

```diff
@@ -270,36 +300,137 @@ function readCSVFile(file, handleQuotes = true) {
     });
 }
 
-let sharedMatrixChart, sharedTreeChart;
-const sharedMatrixData = {}; // Store data grouped by the number of cells for SharedMatrix
-const sharedTreeData = {}; // Store data grouped by the number of cells for SharedTree
+let creationTimeChart, creationMemoryChart, updateTimeChart, updateMemoryChart, removeTimeChart, removeMemoryChart;
```

Comment:
Since these are shared across functions, would be good to document what each is.

Reply:
Added doc